package.json

```diff
@@ -21,9 +21,9 @@
     ]
   },
   "scripts": {
-    "prepare": "lerna link && lerna run prepare",
-    "test": "jest",
-    "lint": "eslint .",
+    "prepare": "genversion --es6 --semi packages/core/src/version.ts && lerna link && lerna run prepare",
```

Comment:
isn't it enough to have `genversion` at the `prepare` script only, which is the stage of the lifecycle that is executed with every command? https://docs.npmjs.com/cli/v8/using-npm/scripts#prepare-and-prepublish

Reply:
Nope, according to the doc, `prepare` is not called before the `test` task (only the `pretest` and `posttest` are called), and lint is not a standard NPM task either so it won't be called there.

Reply:
I mean both `test` and `lint` require having `npm install` on a raw repo, so that `prepare` is executed anyway. But can keep like that as well.